style.css

```diff
@@ -0,0 +1,54 @@
+header {
```

Comment:
- [ ] [OPTIONAL] Please consider styling the page as close as possible to the Figma design including margins/paddings, and sizes

|Your design|Figma design|
|-|-|
|![Screenshot_20220527_210402](https://user-images.githubusercontent.com/25227022/170784270-f50a9e10-f745-4f94-939e-49183c4e1352.png)|![Screenshot_20220527_210527](https://user-images.githubusercontent.com/25227022/170784288-caf84e55-365e-4125-8750-8ab911f91280.png)|

Reply:
Please have in mind that this is a movile phone project.
![image](https://user-images.githubusercontent.com/103516637/170791318-67bb7b3e-c340-478f-aa7a-05ff11ce9e7d.png)